Note to create a service if you extend ExceptionController

pamuche opened the pull request:
I had some trouble to get the custom exception handling with inheritance of ExceptionController working, because I hadn't a clue to make a service out of the ExceptionController of the TwigBundle.

pamuche committed:
Note to create a service if you extend ExceptionController

I had some trouble to get the custom exception handling with inheritance of ExceptionController working, because I hadn't a clue to make a service out of the ExceptionController of the TwigBundle.

javiereguiluz commented on cookbook/controller/error_pages.rst:
```diff
@@ -251,7 +251,25 @@ will be passed two parameters:
+.. note::
+  
+  In case of extending the :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController` 
+  you may configure a service to pass the twig-Environment and the debug-Flag to the constructor. 
```
I would replace this:

```
twig-Environment and the debug-Flag
```

by this:

```
Twig environment and the ``debug`` flag
```

javiereguiluz commented on cookbook/controller/error_pages.rst:
```diff
@@ -251,7 +251,25 @@ will be passed two parameters:
+  
+          # app/config/config.yml
+          services:
+            my_custom_exception_controller_service:
```
In modern Symfony apps, we recommend to use shorter service names. What about replacing `my_custom_exception_controller_service` by `app.exception_controller` ?

javiereguiluz commented:
@pamuche thanks for your contribution. We love comments from people who followed the docs and faced problems or didn't know how to continue. I've left some comments when reviewing your changes, but overall they look OK to me. Thanks!

pamuche committed:
Integrate recommended changes

Thank you for your recommendations. It helps me to contribute better additions to the docs in the future.

pamuche committed:
Adds one forgotten change

WouterJ commented:
Thank you @pamuche for creating this PR and your quick response to the review!

I've merged your pull request into the 2.3 version of the docs (that's the oldest version we maintain) and will merge it up to master from there. After the merge, I've fixed indentation (4 spaces) and added the XML and PHP format for the service in 3b0ea60. Feel free to comment if you have any more questions.
